src/main/scala/mesosphere/marathon/core/task/tracker/TaskTrackerModule.scala

```diff
@@ -0,0 +1,40 @@
+package mesosphere.marathon.core.task.tracker
+
+import akka.actor.ActorRef
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.task.tracker.impl._
+import mesosphere.marathon.metrics.Metrics
+import mesosphere.marathon.state.TaskRepository
+
+/**
+  * Provides the interfaces to query the current task state ([[TaskTracker]]) and to
+  * update the task state ([[TaskUpdater]], [[TaskCreator]]).
+  */
+class TaskTrackerModule(
+    clock: Clock,
+    metrics: Metrics,
+    config: TaskTrackerConfig,
+    leadershipModule: LeadershipModule,
+    taskRepository: TaskRepository) {
+  lazy val taskTracker: TaskTracker = new TaskTrackerDelegate(Some(metrics), config, taskTrackerActorRef)
+
+  def taskUpdater: TaskUpdater = taskTrackerCreatorAndUpdater
+  def taskCreator: TaskCreator = taskTrackerCreatorAndUpdater
```

Comment:
I don't like the name either. But, unfortunately, I cannot think of something better. `TaskCreationListener` sounds to passive. The methods in `TaskCreator` do result in the task being "created/stored" in the `TaskTracker` or a task being removed from the TaskTracker.

Better suggestions?

Maybe we should keep the renaming to another PR, though.